tests/test_filterLabel.py

```diff
@@ -0,0 +1,155 @@
+# This file is part of afw.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import copy
+import unittest
+
+import lsst.utils.tests
+
+from lsst.afw.image import FilterLabel, getDatabaseLabel
+
+
+class FilterLabelTestCase(lsst.utils.tests.TestCase):
+    def setUp(self):
+        self.className = "FilterLabel"
+        self.physicalName = "k-0324"
+        self.band = "k"
+
+    def _labelVariants(self):
+        return iter({FilterLabel(physical=self.physicalName, band=self.band),
+                     FilterLabel.fromBand(self.band),
+                     FilterLabel(band=self.band),
```

Comment:
These two lines produce identical result, right? From `_labelVariants` name not clear if you want tree different varaints or duplicates are OK.

Reply:
They *should* produce identical results, but during development there was a bug that made them inconsistent. I can add a comment.